<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-02HURVS" failures="1" tests="4" name="Test" time="93.066" errors="0" timestamp="2024-04-12T16:47:46 IST">
  <testcase classname="user.CreateNewUserAccountTest" name="createNewUserAccountTest" time="13.35">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at user.CreateNewUserAccountTest.createNewUserAccountTest(CreateNewUserAccountTest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- createNewUserAccountTest -->
  <testcase classname="admin.AdminLoginLogoutTest" name="adminLoginLogoutTest" time="2.563"/>
  <testcase classname="admin.AdminLoginLogoutTest" name="insertProdctAndOrderSameProductTest" time="25.954"/>
  <testcase classname="admin.AdminLoginLogoutTest" name="changeStatusOfOrderedProductTest" time="33.235"/>
  <testcase classname="genericUtilities.BaseClass" name="@AfterSuite closeDBConnection" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite closeDBConnection -->
  <testcase classname="genericUtilities.BaseClass" name="@BeforeSuite connectionDB" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite connectionDB -->
</testsuite> <!-- Test -->
